### DISCUSSION TITLE
Can you increase overall font size?

### DISCUSSION THREAD

**Discussion by HbotondS** [AlmasB/FXGL#1221] [Q&A]
Is there a way to increase font size for **dialogs** and **notifications**? I find the font size a little small, and I didn't figured it out yet if it's possible to increase it or not.

**Comment by HbotondS** [comment]
I solved the problem by implementing my own DialogFactoryService, but it's just too much trouble for only changing the font size. Would be great if we could configure this separately in _initSettings_  maybe.

**Reply by AlmasB** [reply]
Thanks, sounds like something that can be implemented via some UI font size multiplier... I'll turn this into an issue.
